experimental/dds/tree2/src/test/feature-libraries/defaultFieldKinds.spec.ts

```diff
@@ -115,461 +114,285 @@ const childComposer1_2 = (changes: TaggedChange<NodeChangeset>[]): NodeChangeset
 	return nodeChange3;
 };
 
-describe("Value field changesets", () => {
-	const fieldHandler: FieldChangeHandler<
-		FieldKindsTypes.ValueChangeset,
-		FieldKinds.ValueFieldEditor
-	> = FieldKinds.value.changeHandler;
-
-	const change1WithChildChange: FieldKindsTypes.ValueChangeset = {
-		value: { set: tree1 },
-		changes: nodeChange1,
-	};
-	const childChange1: FieldKindsTypes.ValueChangeset = { changes: nodeChange1 };
-	const childChange2: FieldKindsTypes.ValueChangeset = { changes: nodeChange2 };
-	const childChange3: FieldKindsTypes.ValueChangeset = { changes: nodeChange3 };
-
-	const change1 = tagChange(fieldHandler.editor.set(singleTextCursor(tree1)), mintRevisionTag());
-	const change2 = tagChange(fieldHandler.editor.set(singleTextCursor(tree2)), mintRevisionTag());
-
-	const revertChange2: FieldKindsTypes.ValueChangeset = {
-		value: {
-			revert: singleTextCursor(tree1),
-			changeId: { revision: change2.revision, localId: brand(42) },
-		},
-	};
-
-	const simpleChildComposer = (changes: TaggedChange<NodeChangeset>[]) => {
-		assert.equal(changes.length, 1);
-		return changes[0].change;
-	};
-
-	it("can be created", () => {
-		const expected: FieldKindsTypes.ValueChangeset = { value: { set: tree1 } };
-		assert.deepEqual(change1.change, expected);
-	});
-
-	it("can be composed", () => {
-		const composed = fieldHandler.rebaser.compose(
-			[change1, change2],
-			simpleChildComposer,
-			idAllocator,
-			crossFieldManager,
-			defaultRevisionMetadataFromChanges([change1, change2]),
-		);
-
-		assert.deepEqual(composed, change2.change);
-	});
-
-	it("can be composed with child changes", () => {
-		const taggedChildChange1 = tagChange(childChange1, mintRevisionTag());
-		assert.deepEqual(
-			fieldHandler.rebaser.compose(
-				[change1, taggedChildChange1],
-				simpleChildComposer,
-				idAllocator,
-				crossFieldManager,
-				defaultRevisionMetadataFromChanges([change1, taggedChildChange1]),
-			),
-			change1WithChildChange,
-		);
-
-		const expected: FieldKindsTypes.ValueChangeset = {
-			value: { set: tree1 },
-			changes: nodeChange1,
-		};
-
-		assert.deepEqual(change1WithChildChange, expected);
-		assert.deepEqual(
-			fieldHandler.rebaser.compose(
-				[makeAnonChange(childChange1), change1],
-				simpleChildComposer,
-				idAllocator,
-				crossFieldManager,
-				defaultRevisionMetadataFromChanges([change1]),
-			),
-			change1.change,
-		);
+describe("Optional and Value field changesets", () => {
```

Comment:
I like your spirit of deduplication, but since the contract for the two field kinds remains slightly different, we need to make sure the tests capture that difference. That's not to say *some* of the test code couldn't be deduplicated, but I don't think you have worry about it as part of this PR, up to you. :)

Reply:
I went back and re-added the value field tests. I left a comment about one that gave me trouble, but if you could take a look to make sure that I correctly translated the tests in general that would be great!